core/domain/skill_services_test.py

```diff
@@ -1289,6 +1289,31 @@ def test_create_multi_user_skill_mastery(self):
         self.assertEqual(
             degrees_of_mastery, {skill_id_4: 0.3, skill_id_5: 0.5})
 
+    def test_get_sorted_skill_ids(self):
```

Comment:
This test is too complicated, can you test `get_sorted_skill_ids` directly?

Reply:
@vojtechjelinek  In `get_sorted_skill_ids` we have to check for whether the function is returning the first`feconf.MAX_NUMBER_OF_SKILL_IDS` or not. For that I need to create more than 20 entry in the dict. I'm not sure how to do that here. Please suggest some alternative.

Reply:
Either create it manually or generate it with some for-loop.